frontend/benefit/applicant/src/hooks/useApplicationQuery.ts

```diff
@@ -16,7 +16,12 @@ const useApplicationQuery = (
         : handleResponse<ApplicationData>(
             axios.get(`${BackendEndpoint.APPLICATIONS}${id}/`)
           ),
-    { enabled: Boolean(id), retry: false }
+    {
```

Comment:
Not sure if the disabling is the best option, refetching has good things also. In react-hook-form I used reset-function to keep the data on form: 
https://github.com/City-of-Helsinki/yjdh/blob/03d1819671203094d8ad4d0432c9f981b76ea8fc/frontend/kesaseteli/employer/src/components/application/ApplicationForm.tsx#L21
and 
https://github.com/City-of-Helsinki/yjdh/blob/develop/frontend/kesaseteli/employer/src/hooks/application/useResetApplicationFormValues.ts
Not sure what's equivalent on formik, is it this one: https://formik.org/docs/api/formik#resetform-nextstate-partialformikstatevalues--void